tests/snap/comments.test.scala

```diff
@@ -3,6 +3,7 @@
 
 //> using packaging.version "1.0.0"
 //> using scala 2.13
+//> using dep org.scala-lang::toolkit:0.2.0
```

Comment:
Could you also add cases like:
```
 using deps org.scala-lang::toolkit:0.2.0,org.scala-lang::toolkit:0.2.1
 using numbers 123, 111
 using strings "asdf", bfds 
```

Reply:
I added more tests. There is still an issue with case like
```
//> using negative -10
```
as mentioned here https://github.com/scala/vscode-scala-syntax/issues/253, because `#constants` doesn't include negative numbers, but changing the order of patterns allowed correct highlighting of booleans